frontends/web/src/components/amount/conversion-amount.tsx

```diff
@@ -70,12 +69,8 @@ export const ConversionAmount = ({
             unit={conversionUnit}
           />
         </>
-      ) : (
-        <div className={styles.skeletonContainer}>
-          <Skeleton />
-        </div>
-      )}
-      <span className={styles.txUnit}>{' '}{conversionUnit}</span>
+      ) : null }
```

Comment:
cc @thisconnect @shonsirsha ack on this? 

I think that we should use skeletons only while waiting for backend responses, while here the data is just missing.